<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Kozak</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Birthday reminder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/birth.ico</normaloff>../Icons/birth.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1E1E1E;
}
QPushButton{
	background-color: black;
	color: yellow;
	border: 1px solid #483D8B;
}
QPushButton:pressed{
	background-color: #2F4F4F;
	color: #FF0000;
	border: 1px dashed #000080;
}
QPushButton:hover{
	border: 2px dashed #00FA9A;
	color: #00FA9A;
}
QPushButton:disabled{
	background-color: black;
	color: #800000;
	border: none;
}

QLineEdit{
	background-color: #DCDCDC;
	color: 	#000080;
	border: 1px solid yellow; 
}
QLineEdit:focus{
	border: 3px dashed #800000;
}
QLabel{
	color: #FF4500;
}
QGroupBox{
	backgrond-color: #696969;
	color: #7FFF00;
	border: 1px solid #0000CD;
}
QRadioButton{
	color: #FF4500;
}
QRadioButton:hover{
	color: #00FF00;
}

QCalendarWidget QWidget{
	alternate-background-color: #008B8B;	
}
QCalendarWidget QAbstractItemView:enabled{
	background-color: #000;
	color: #FFD700;
	border: 1px solid #8A2BE2;
	selection-background-color: #2F4F4F;
	selection-color: rgb(0, 255, 0);
}

#qt_calendar_navigationbar{
	background-color: #2F4F4F;
	border: 1px solid #8A2BE2;
} 
#qt_calendar_prevmonth,
#qt_calendar_nextmonth{
	border: none;
	qproperty-icon: none;
	min-width: 13px;
	max-width: 13px;
	min-height: 13px;
	max-height: 13px;
	background-color: transparent;
	padding: 5px;
	color: #00FF00;
}
#qt_calendar_prevmonth{
	matgin-left: 5px;
	qproperty-text: &quot;&lt;&quot;;
}
#qt_calendar_nextmonth{
	matgin-right: 5px;
	qproperty-text: &quot;&gt;&quot;;
}
#qt_calendar_prevmonth:hover,
#qt_calendar_nextmonth:hover{
	border: 1px dashed #00FA9A;
	color: #00FA9A;
    background-color: #000;
}
#qt_calendar_prevmonth:pressed,
#qt_calendar_nextmonth:pressed{
	border: 1px dashed #00FA9A;
	color: #00FA9A;
    background-color: #000;
}
#qt_calendar_yearbutton{
	background-color: #2F4F4F;
	color: #00FA9A;
	margin: 5px;
	border_radius: 5px;
	font-size: 13px;
	padding: 5px 6px;
}
#qt_calendar_monthbutton{
	background-color: #2F4F4F;
	color: #00FA9A;
	width: 110px;
	margin: 4px;
	border_radius: 5px;
	font-size: 13px;
	padding: 5 px 8px;
}
#qt_calendar_yearbutton:hover,
#qt_calendar_monthbutton:hover{
    border: 2px dashed #00FA9A;
	color: #00FA9A;
    background-color: #000;	
}
#qt_calendar_yearbutton:pressed,
#qt_calendar_monthbutton:pressed{
    border: 2px dashed #00FA9A;
	color: #00FA9A;
    background-color: #000;	
}
QCalendarWidget QToolButton QMenu{
	backgrond-color: #2F4F4F;
	color: yellow;
}
QCalendarWidget QToolButton QMenu:item{
	border: 1px solid red;
	padding: 5px;
}
QCalendarWidget QToolButton QMenu:item:selected{
	border: 1px dashed #00FA9A;
	color: #00FA9A;
}

QListWidget{
	border : 1px solid #8A2BE2;
	background : #696969;
}
QListView::item{
	border : 1px solid #FF7F50;
	color: yellow;
	background : #00008B;
	padding: 5px;
	font-size: 13px;
}</string>
  </property>
  <property name="locale">
   <locale language="Ukrainian" country="Ukraine"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>312</width>
      <height>211</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="locale">
     <locale language="Ukrainian" country="Ukraine"/>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>220</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Шлях до файлу</string>
    </property>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Шлях...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>96</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_path_file">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_path">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>55</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_exists_file">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>65</y>
       <width>166</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Дії з даними</string>
    </property>
    <widget class="QPushButton" name="pushButton_create_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Створити дані</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_update_file">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Змінити дані</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>160</y>
      <width>239</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFD700;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>204</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Таймер до закриття додатку</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>144</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: yellow;</string>
     </property>
     <property name="text">
      <string>До закриття додатку залишилось:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_timer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>48</y>
       <width>144</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: yellow; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop_timer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Зупинити закриття</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>305</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вихід</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setting">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>305</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Налаштування</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupBox</sender>
   <signal>clicked()</signal>
   <receiver>line_path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
